EncoreTickets.SDK/Inventory/Models/RequestModels/SeatAvailabilityParameters.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using EncoreTickets.SDK.Api.Models;
+
+namespace EncoreTickets.SDK.Inventory.Models.RequestModels
+{
+    /// <summary>
+    /// The helper entity for collecting parameters for obtaining seat availability.
+    /// </summary>
+    public class SeatAvailabilityParameters
+    {
+        /// <summary>
+        /// Gets or sets performance date: if nothing is sent, current date will be used.
+        /// </summary>
+        public DateTime? Date { get; set; }
+
+        /// <summary>
+        /// Gets or sets performance time: if nothing is sent, current time will be used.
+        /// </summary>
+        public DateTime? Time { get; set; }
```

Comment:
I'm not sure, because, as I remember, sometimes we need to adjust the time, for example, from a separate datetime structure, and this is convenient using the current object structure for parameters. And also the properties of this object correspond exactly to arguments that the API receives.

Reply:
You seem to be copying those parameters to another object anyway so I don't see why we can't do it from a common DateTime struct. Also, could you please clarify in which cases we need to adjust the time and why we can't do it with a common DateTime struct? If there is some reason why we actually need this, could you please also clarify why we use DateTime instead of TimeSpan for time?

Reply:
Maybe you are right because at present I can not find where we need to separate these values) I will fix it